astropy/modeling/tests/test_compound.py

```diff
@@ -307,6 +307,34 @@ def test_compound_with_polynomials(poly):
     assert_allclose(result, result_compound)
 
 
+def test_variable_substitution():
```

Comment:
Please rename this function to `test_fix_inputs` os it's easy to discover. Likewise the one below.

Reply:
done